### ISSUE TITLE
Bigsur installation problem

### ISSUE THREAD

**Issue by ptuchster** [MadRabbit/halmak#23]
On big sur 11.01 only few apps allow me to use halmak as input source, such as terminal and different ides. For example in Safari I am unable to select halmak as source even id it's appear in every select option.

**Comment by janniksinz** [comment]
Yes that is a big problem! It automatically switches the input source. No warning (not that it would make it better).
Version 11.1 (20C69)

**Comment by bortrenamo** [comment]
Same issue here.

**Comment by dimaqq** [comment]
I had similar problem with anther layout, Workman, but only until reboot. In the end I did both:
* copy .keyboard and .icns files directly into `/Library/Keyboard\ Layouts/`
* reboot

E voilà, Safari worked with the new layout again.
(YMMV)

**Comment by flakedev** [comment]
@dimaqq please can you explain a little more about how you copied the files without the bundle? I'm finding that opening any new app switches my keyboard back to Qwerty British.

**Comment by dimaqq** [comment]
Sure, I took individual files from https://github.com/MadRabbit/halmak/tree/master/macos/Halmak.bundle/Contents/Resources that is from inside the bundle, and dropped them into the system dir, with the following result

```
> ls -hl /Library/Keyboard\ Layouts/
total 84K
-rw-r--r-- 1 root wheel 38K Jan 18 19:06 Halmak.icns
-rw-r--r-- 1 root wheel 41K Jan 18 19:06 Halmak.keylayout
```

**Comment by dimaqq** [comment]
Oh and I had to reboot the laptop after adding the files. Without reboot I had the same issue: some 1st party (Safari) kept reverting to a standard keyboard automatically.

**Comment by kaievns** [comment]
hey folks, thanks for reporting the issue. unfortunately im not using OSX anymore, so i cant really fix this. if anyone makes a patch please make a PR, i'd merge that happily

**Comment by danielpcox** [comment]
I had to install it, reboot, add it to my available layouts, and then reboot again, but after that it worked for me under Monterey 12.3. I also had to put the bundle in `~/Library/Keyboard\ Layouts/` rather than the global one.
